### PR TITLE
NEXUS-24692: Fix broken API after changes on the nexus-internal

### PR DESCRIPTION
**Pull request by aornatovskyy**
Ticket: https://issues.sonatype.org/browse/NEXUS-24692
CI: https://jenkins.ci.sonatype.dev/job/nxrm/job/nxrm3/job/nxrm-community-it/job/NEXUS-24681_nxrm_jenkins_shared_in_community_it/11/consoleFull

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.25.1-03</version>
+    <version>3.26.0-SNAPSHOT</version>
```
Comment (Mykyta) [inline pom.xml:22]: Will it be changed to 3.26.0?

**Comment by Mykyta** [comment]
For the classes with @Deprecated annotation add pls class header with Description:
add 

`@deprecated the 'beta' prefix is being phased out, prefer starting new APIs with {@link APIConstants#V1_API_PREFIX} instead. `

File: pom.xml (hunk 1 above)
Comment (aornatovskyy) [reply pom.xml:22]: yes exactly

File: nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java [hunk 2]
```diff
@@ -80,7 +80,6 @@ public void testUploadFailedWrongPackageExtension() throws Exception
   {
     HttpResponse httpResponse = uploadSinglePackage(AGRICOLAE_131_XXX);
     assertThat(httpResponse.getStatusLine().getStatusCode(), is(BAD_REQUEST));
```
Comment (sergeyonofreychuck) [inline nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java:82]: we need to investigate the problem. Please create a technical debt ticket

Comment (aornatovskyy) [reply nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java:82]: https://issues.sonatype.org/browse/NEXUS-24838

**Pull request merged**